_data/testingcodeswanlake.yml

```
@@ -7,45 +7,72 @@
         - title: Writing a Simple Function
           url: /learn/testing-ballerina-code/quick-start-on-testing/#writing-a-simple-function
           active: writing-a-simple-function
-        - title: What’s Next?
-          url: /learn/testing-ballerina-code/quick-start-on-testing/#whats-next
-          active: whats-next
     - title: Writing Tests
       url: '#'
       sublinks:
-        - title: Project Structure
-          url: /learn/testing-ballerina-code/writing-tests/#project-structure
-          active: project-structure
-        - title: Test Source Files
-          url: /learn/testing-ballerina-code/writing-tests/#test-source-files
-          active: test-source-files
-        - title: Test Resources
-          url: /learn/testing-ballerina-code/writing-tests/#test-resources
-          active: test-resources
         - title: Defining a Test
```

```suggestion
        - title: Defining a Test
```
This should also be `Defining Tests` to make the titles consistent.